analytics/management/commands/update_analytics_tables.py

```diff
@@ -0,0 +1,69 @@
+from argparse import ArgumentParser
+from datetime import timedelta
+
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from analytics.lib.data_collectors import AnalyticsStat, process_count
+from analytics.models import RealmCount, UserCount, StreamCount, DeploymentCount, PMCount, HuddleCount
+from zerver.lib.timestamp import string_to_datetime as sd, datetime_to_string as ds
+from analytics.lib.interval import timeinterval_range
+from zerver.models import UserProfile, Message, Stream
+
+
+class Command(BaseCommand):
+    help = """Fills RealmCount table.
+
+    Run as a cron job that runs every hour."""
+
+    def add_arguments(self, parser):
+        # type: (ArgumentParser) -> None
+        parser.add_argument('-start', type=str,
```

Comment:
I think -s and --start are more conventional (and we should allow both). The convention is one dash for one character args, and two dashes for multicharacter args, which allows things like -xyz where x, y, and z are each one character arguments.